src/modules/universal-discovery/components/finder/css/finder.component.css

```diff
@@ -7,7 +7,6 @@
     flex-wrap: nowrap;
     height: 100%;
     overflow: auto;
-    border-radius: 4px;
```

Comment:
Removing this property fixes bug. It seems that it was too much for Firefox to render properly. 🤷‍♂️ 
It was actually redundant property, removing it does not change UI.

Reply:
Bug was appearing when some folder had enough items for scroll to appear in _.c-finder__branch_ div. Everything seems to be related with bug in Firefox browser which was unable to handle too many scrolls, overflows and border-radiuses.